### ISSUE TITLE
Let users pick the post's revision.

### ISSUE THREAD

**Issue by diegoreymendez** [wordpress-mobile/WordPress-iOS#7810]
Users should have an option to pick the revision of a post they want to edit.

/cc @iamthomasbishop - As this is related to #5686 and #7809.

**Comment by designsimply** [comment]
Closing as fixed. 

I tested with the following steps and was able to pick a revision I wanted to edit:

1. Open any post with several revisions using the Aztec (or block) editor in the app.
1. Go to More (top right) > History.
1. Select any revision.
1. Tap "Load" at top right.

**Result:** I am able to load recent revisions for a post I am editing either with the Aztec or block editor.

<img src="https://user-images.githubusercontent.com/1119271/70093346-6c784b00-15dd-11ea-9cf4-4423c0521c8c.PNG" width="200" alt="IMG_3155"> <img src="https://user-images.githubusercontent.com/1119271/70093347-6d10e180-15dd-11ea-9f24-ab680abd367e.PNG" width="200" alt="IMG_3156"> <img src="https://user-images.githubusercontent.com/1119271/70093349-6d10e180-15dd-11ea-9e18-c53bccfa8d44.PNG" width="200" alt="IMG_3157">
<sup>Tested with WP Internal 13.8.0.20191202 MS App Center beta on iPhone 6S iOS 13.2.3.</sup>

@iamthomasbishop no action needed if I've got the right steps above for testing this, but I also wanted to check with you to see if anything is missing in this flow or if I'm closing this issue prematurely?
